cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/TypeBuilder.kt

```diff
@@ -25,32 +25,29 @@
  */
 package de.fraunhofer.aisec.cpg.graph
 
+import de.fraunhofer.aisec.cpg.graph.types.NumericType
+import de.fraunhofer.aisec.cpg.graph.types.NumericType.Modifier
 import de.fraunhofer.aisec.cpg.graph.types.ObjectType
-import de.fraunhofer.aisec.cpg.graph.types.ObjectType.Modifier
-import de.fraunhofer.aisec.cpg.graph.types.Type
 
 fun MetadataProvider.newPrimitiveType(
```

Comment:
Extend it from LangaugProvider instead of MetadataProvider?

Reply:
Yes